src/KubernetesClient.VanillaRest/VanillaKubernetes.cs

```diff
@@ -0,0 +1,74 @@
+using System.IO;
+using System.Net;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using k8s.Autorest;
+using System.Net.Http.Headers;
+
+
+namespace k8s
+{
+    public abstract partial class VanillaKubernetes
```

Comment:
I'd prefer a more descriptive name than 'Vanilla' so that it is understandable to someone browsing the code.

Reply:
any ideas? I update the PR with details proposal, did not have a better name in mind.

Reply:
ping @brendandburns

Reply:
Apologies for the delay, perhaps `AbstractKubernetes` or `BasicKubernetes`?